### DISCUSSION TITLE
recording SIG Registries meetings

### DISCUSSION THREAD

**Discussion by ricochet** [bytecodealliance/SIG-Registries#46] [Polls]
In the last SIG Registry meeting, we discussed the idea of recording meetings. Many organizations that develop around open source communities consider recording meetings part of developing in the open, see the [Kubernetes Zoom Guidelines](https://github.com/kubernetes/community/blob/master/communication/zoom-guidelines.md#meeting-recordings) for an example. This poll is meant to open discussion for how folks in this group feel about recording our meetings.

**Comment by ricochet** [comment]
I personally want to record and make the recording public for the following reasons:
- I am looking for ways to we can better include other timezones
- There are several free transcript tools that we can use. We should still take notes on decisions and high-level items, but can lean on a transcript for details.
- I'd like to use Youtube Live to make uploading to Youtube automatic so it shouldn't add extra work

**Reply by esoterra** [reply]
The [w3c process](https://www.w3.org/2021/Process-20211102/#meeting-recording) has a strict consent-focused policy for meeting recording
> No-one may take an audio or video recording of a meeting, or retain an automated transcript, unless the intent is announced at the start of the meeting, and no-one participating in the recorded portion of the meeting withholds consent. If consent is withheld by anyone, recording/retention must not occur. The announcement must cover: (a) who will have access to the recording or transcript and (b) the purpose/use of it and (c) how it will be retained (e.g. privately, in a cloud service) and for how long.

What kind of policy would we have?

**Reply by ricochet** [reply]
I like how the CNCF handles this: At the beginning of every meeting, they announce that the meeting is being recorded and communications are subject to the [bytecode alliance code of conduct](https://github.com/bytecodealliance/governance/blob/main/CODE_OF_CONDUCT.md). In the meetings repo we would add details around the process and where the recordings are kept on youtube. Some examples of how this is documented in other communities:
- [kubernetes meeting-recordings](https://github.com/kubernetes/community/blob/master/communication/zoom-guidelines.md#meeting-recordings)
- [kubernetes youtube guidelines](https://github.com/kubernetes/community/blob/master/communication/youtube/youtube-guidelines.md)
